models/public/yolo-v2-tiny-tf/accuracy-check.yml

```diff
@@ -5,7 +5,7 @@ models:
       - framework: tf
         tags:
           - FP32
-        model:   yolo-v2-tiny.pb
+        model: yolov2-tiny.pb
```

Comment:
I don't see any use in such renaming since you can specify the output name in the script below.

Reply:
Well, I originally changed the name to align it with the name of the weights file. But I suppose that is a gratuitous change. Reverted.